app/client/sagas/exportSaga.js

```diff
@@ -9,30 +9,27 @@ import {
   cancel,
   race
 } from "redux-saga/effects";
+import { handleError } from "../utils";
 
-import {
-  NEW_EXPORT_REQUEST,
-  NEW_EXPORT_SUCCESS,
-  NEW_EXPORT_FAILURE
-} from "../reducers/exportReducer.js";
+import { ExportAction } from "../reducers/export/actions";
+import { ExportActionTypes } from "../reducers/export/types";
 
 import { exportAPI } from "../api/exportAPI";
 
 function* newExport({ payload }) {
   try {
     const result = yield call(exportAPI, payload);
-    if (result.status === "success") {
```

Comment:
a great PR would be to
CMD+F `== "`  and remove any string comparisons where possible

Reply:
yeah, I actually think this is the last one in a reducer